src/socketio.quiver.js

```diff
@@ -417,6 +503,13 @@ QuiverSocialProvider.prototype.connectAsOwner_ = function(server, continuation)
 
     // Connect to self, in order to be able to send messages to my own other clients.
     this.connectAsClient_(this.configuration_.self, null, server, continuation);
+
+    // Add the server to our public list of contact points.
+    this.addServer_(server);
```

Comment:
We should move this above the emits which invoke this.makeIntroMsg_.  Right now, if we just connected to a server that wasn't part of the initial configuration, ```this.configuration_.self.servers_``` will be missing the latest server.  That will result in makeIntroMsg_ adding an ```undefined``` to the intro message, which later results in JS exceptions